File: OPENAPI-REST-API/swagger-client/spring/src/main/java/com/interserver/myadmin/model/IdVerifyBody1.java
package com.interserver.myadmin.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.v3.oas.annotations.media.Schema;
import org.springframework.validation.annotation.Validated;
import org.openapitools.jackson.nullable.JsonNullable;
import io.swagger.configuration.NotUndefined;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonSetter;
import com.fasterxml.jackson.annotation.Nulls;
import javax.validation.Valid;
import javax.validation.constraints.*;

/**
 * IdVerifyBody1
 */
@Validated
@NotUndefined



public class IdVerifyBody1   {
  @JsonProperty("cc_ccv2")

  private String ccCcv2 = null;


  public IdVerifyBody1 ccCcv2(String ccCcv2) { 

    this.ccCcv2 = ccCcv2;
    return this;
  }

  /**
   * The CVV/CVC code on the back of the credit card.
   * @return ccCcv2
   **/
  
  @Schema(required = true, description = "The CVV/CVC code on the back of the credit card.")
  
  @NotNull
  public String getCcCcv2() {  
    return ccCcv2;
  }



  public void setCcCcv2(String ccCcv2) { 

    this.ccCcv2 = ccCcv2;
  }

  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    IdVerifyBody1 idVerifyBody1 = (IdVerifyBody1) o;
    return Objects.equals(this.ccCcv2, idVerifyBody1.ccCcv2);
  }

  @Override
  public int hashCode() {
    return Objects.hash(ccCcv2);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class IdVerifyBody1 {\n");
    
    sb.append("    ccCcv2: ").append(toIndentedString(ccCcv2)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/id_verify_body1.dart
part of swagger.api;

class IdVerifyBody1 {
  /* The CVV/CVC code on the back of the credit card. */
  String ccCcv2 = null;

  IdVerifyBody1();

  @override
  String toString() {
    return 'IdVerifyBody1[ccCcv2=$ccCcv2, ]';
  }

  IdVerifyBody1.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    ccCcv2 = json['cc_ccv2'];
  }

  Map<String, dynamic> toJson() {
    return {
      'cc_ccv2': ccCcv2
     };
  }

  static List<IdVerifyBody1> listFromJson(List<dynamic> json) {
    return json == null ? new List<IdVerifyBody1>() : json.map((value) => new IdVerifyBody1.fromJson(value)).toList();
  }

  static Map<String, IdVerifyBody1> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, IdVerifyBody1>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new IdVerifyBody1.fromJson(value));
    }
    return map;
  }
}
